index.bs

```diff
@@ -801,11 +807,32 @@ expected that an object that implements a particular IDL interface
 provides ways to inspect and modify the object's state and to
 invoke the behavior described by the interface.
 
+[=Interfaces=] and [=partial interfaces=] have an <dfn for="interface,partial interface">enclosing
+module</dfn>, which is a [=module=] or null.
+Unless otherwise specified, it is null.
+
+<div algorithm>
+  [=Interfaces=] and [=partial interfaces=] have a
+  <dfn for="interface,partial interface">scoped identifier</dfn>,
+  which uniquely identifies an [=interface=].
```

Comment:
Why is this a separate concept from qualfiied name? The difference seems to be that for modules we allow two same-named interfaces in different modules, but we don't allow two same-named interfaces in [LegacyNamespace]. Is that mismatch desirable?

Reply:
Changing how LegacyNamespace works in this PR seems out of scope to me. Note that, if I connected everything correctly, using the qualified name would mean you'd need changes like this to specifications that use it:

```diff
 namespace WebAssembly {
-    Promise<Module> compile(BufferSource bytes);
+    Promise<WebAssembly.Module> compile(BufferSource bytes);
 };
```

Maybe we want that, but let's discuss that separately.